recipes/powerfit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "powerfit" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Any reason to not use the PyPI URL?

Reply:
Yes, GitHub is the original source, so this is the highest likeliness that the package stays up-to-date

Reply:
GH tags are not static and can lead to changed hashes. We are not against using them but it may be a headache sometimes. Also, some folks don't follow good practices when tagging and that may also cause headaches. PyPI kinds of filter those out for us.